cmd/guacone/cmd/vulnerability.go

```diff
@@ -50,14 +51,28 @@ type queryOptions struct {
 	vulnerabilityID string
 	depth           int
 	pathsToReturn   int
+	inputType       string
 }
 
 var queryVulnCmd = &cobra.Command{
-	Use:   "vuln [flags] <purl/sbomURI>",
+	Use:   "vuln <type> <input>",
```

Comment:
If I'm reading it right, this is a compatability-breaking change in the CLI. We should make sure any docs and demos that use the `vuln` command are updated accordingly.

Alternatively, is there a way we can figure out from the `<input>` what the `<type>` is so that the user doesn't have to specify it? That would, I think, preserve compatibility.

Reply:
@funnelfiasco, thank you for the reminder, I forgot to change the docs. So, I have created a PR to the docs https://github.com/guacsec/guac-docs/pull/150.

Additionally, I have no idea how to figure out the type from the input because of these couple reasons. If you know out a way to get around these issues, then please let me know, and I will update the code!

* I was thinking of first checking whether the input string has a colon in it (":") because artifacts are created in the format of `<"algorithm">:<"digest">`. The issue with this is that purls also have colons in them.
* Then I thought about checking if the input string has a `pkg:` at the beginning. But, SBOM URIs can have that in them as well.

Reply:
Thanks for the docs PR! I'll review shortly. As for detecting the type, I figured that would probably be the answer, but it was worth asking 😄